pylops/basicoperators/block.py

```diff
@@ -7,32 +7,31 @@
 from pylops.utils.typing import DTypeLike
 
 
-def _Block(
-    ops: Iterable[Iterable[LinearOperator]],
-    dtype: Optional[DTypeLike] = None,
-    _HStack=HStack,
-    _VStack=VStack,
-    args_HStack: Optional[dict] = None,
-    args_VStack: Optional[dict] = None,
-) -> LinearOperator:
+class _Block(VStack):
```

Comment:
There is a bit of a subtlety here. The original function definition uses `_HStack` and `_VStack` (arguments) in the function body. The new class-based definition also uses `_HStack` and `_VStack` in the `__init__`, but the class itself is derived from `VStack`. I think it would be more consistent to derive `_Block` from `LinearOperator`, and make the necessary changed to the `super().__init__` call.

Reply:
Done.
Also added this in Block
```
if nproc > 1:
   self.pool = mp.Pool(processes=nproc)
```